charts/pwa/README.md

```diff
@@ -35,10 +35,68 @@ spec:
   values:
 ```
 
-## Upgrading an existing Release to a new Major Version
+### Using Flux v2
 
-A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an
-incompatible breaking change needing manual actions. These actions will be descibed as part of the release documentation available on GitHub.
+Here you create a HelmRepository resource in addition to the HelmRelease [(helm-operator-migration Guide from Flux v1 to Flux v2)](https://fluxcd.io/flux/migration/helm-operator-migration/)
+
+```yaml
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: ish-helm-charts
+  namespace: flux-system
+spec:
+  interval: 1m0s
+  url: https://intershop.github.io/helm-charts
+
+---
+# PWA HelmRelease
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: ${namespace}
+  namespace: ${namespace}
+spec:
+  chart:
+    spec:
+      # pwa helm chart, version from https://github.com/intershop/helm-charts
+      chart: pwa-main
+      version: 0.6.0
+      # Source reference to the HelmChart Repo
+      sourceRef:
+        kind: HelmRepository
+        name: ish-helm-charts
+        namespace: flux-system
+  # in case of multiple pwa instances will be deployed into the given environment namespace, a postfix need to be added the
+  # release name (i.e. pwa-$ENVIRONMENT-01 or pwa-$ENVIRONMENT-edit)
+  releaseName: ${namespace}
+  targetNamespace: ${namespace}
+  interval: 1m0s
+  timeout: 5m0s
+   # Helm Values - to be adapted by the dev team
+  values:
+```
+
+## Release Versions
+
+The following table provides an overview of the different PWA Helm Chart versions and the minimum required PWA version to use it with.
+In addition the version changes and possibly necessary migration information is given.
+
+| Chart | PWA    | Changes                                                                                                                                                                                           | Migration Information                                                                                                   |
+| ----- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| 0.7.0 | 1.0.0  | re-enabled support for `multi-channel.yaml` and `caching-ignore-params.yaml` source code fallbacks<br/>add additional ingress for domain whitelisting<br/>add labels on deployment and pod levels | removed deprecated configuration options<br/>`cache.enabled` - was not optional<br/>`cache.channels`<br/>`upstream.icm` |
```

Comment:
You think it's enough to mention the removal of deprecated customization options here? I have seen the usage of e.g. `upstream.icm` in a lot customer's flux configs. Maybe a migration note would be helpful?

Reply:
I added migration notes for 0.7.0. With this everybody should be able to figure out what to do.